recipes/sklearn-contrib-lightning/run_test.py

```diff
@@ -0,0 +1,8 @@
+import lightning
+import nose
+import os
+
+lightning_dir = os.path.dirname(
+    os.path.abspath(lightning.__file__))
+os.chdir(lightning_dir)
+nose.run(module=lightning)
```

Comment:
I have a personal preference to run the `unittests` like you are doing here. However, sometimes we are time limited by the CIs and you'll have to reduce the scope of the tests, like a simple `import` test.